install-protobuf.sh

```diff
@@ -13,8 +13,13 @@ if [ $? -ne 0 ] ; then
   else
     echo "Install protobuf from web."
     git clone https://github.com/google/protobuf.git
+    # For unknown reason, compiling protobuf on travis-ci breaks with clang.
+    # So we override to always use gcc.
+    CC=gcc
+    CXX=g++
     cd protobuf && git checkout v3.1.0
-    ./autogen.sh && ./configure --prefix=/usr && make && sudo make install && sudo ldconfig
+    ./autogen.sh && ./configure --prefix=/usr && make && \
```

Comment:
This line is getting pretty silly. This section could also be written as:
```bash
(
  # For unknown reason, compiling protobuf on travis-ci breaks with clang.
  # So we override to always use gcc.
  CC=gcc
  CXX=g++
  set -e
  cd protobuf
  git checkout v3.1.0
  /autogen.sh
  ./configure --prefix=/usr
  make
  sudo make install
  sudo ldconfig
)
```

Whether to make that change is a matter of taste which I leave up to you.